### ISSUE TITLE
[A11Y] "Description" form has no visible lable

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3455]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-08-18)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Observe "Learn more about web compatibility" link.

**Expected Behavior:**
Description form has visible label.

**Actual Behavior:**
Description form has no visible label.

**Notes:**
1. Screenshot attached.
![image](https://user-images.githubusercontent.com/12184325/90649620-9a70ee80-e243-11ea-83c8-e2c0fd6b0e55.png)
2. Read more about it here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

**Comment by miketaylr** [comment]
Yeah, good call. We should be using a `<label>` instead of a `placeholder` for "Describe what happened." I guess we could position it with CSS and hide it when the textarea has focus to replicate the current design.
